charts/dapr/charts/dapr_placement/templates/dapr_placement_statefulset.yaml

```diff
@@ -186,6 +186,8 @@ spec:
             valueFrom:
               fieldRef:
                 fieldPath: metadata.namespace
+          - name: PLACEMENT_RAFT_SETTINGS
+            value: v2
```

Comment:
I am allowing users to revert to v1 in case the new setting shows issues at scale.